corehq/ex-submodules/casexml/apps/phone/tests/test_dbaccessors.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+from django.test import TestCase
+from casexml.apps.phone.dbaccessors.sync_logs_by_user import get_sync_logs_for_user, \
+    get_last_synclog_for_user
+from casexml.apps.phone.models import SyncLog, SimplifiedSyncLog
+from corehq.util.test_utils import DocTestMixin
+
+
+class DBAccessorsTest(TestCase, DocTestMixin):
```

Comment:
as far as i can tell this doesn't test the multi-db scenario at all, does it? could be nice to add one more case